kernel-module/build-for-jetson-drive-igpu-native.sh

```diff
@@ -20,4 +20,26 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
+SRC_DIR="$(find /usr/src/nvidia/nvidia-oot/* -name nv-p2p.h 2>/dev/null|head -1|xargs dirname 2>/dev/null)"
```

Comment:
here we want to find the location of the directory that contains these nv-p2p.h and module symbols in doing so there are may be two places i.e. line 23 and line 25. Can we check those places once and for all at the start so you do not have to find it again like in line 34 and 35?

Reply:
I have optimized the build script for iGPU where we find first correct kernel path and then we are checking for header and module symbols.